### PR TITLE
Add read-only inputs to UI plus Copy to Clipboard buttons

### LINKED ISSUE #3066: Make recovery phrase copy/pastable
Currently, the user cannot select/copy/paste the recovery phrase when creating an account.
That is not convenient at all!

### PR DESCRIPTION
**Pull request by ngotchac**
This should fix #3066 #3052 and #3009 

`disabled` inputs are converted to `readonly` inputs.
New props can be added to `Input` components to render a _Copy to Clipboard_ button.
This has been added in Contracts/Accounts/Events/Account Creation....

### CODE REVIEW

File: js/src/views/Contract/Events/Event/event.js [hunk 1]
```diff
@@ -129,10 +129,12 @@ class Event extends Component {
 
         return (
           <Input
-            disabled
+            readOnly
```
Comment (jacogr) [inline js/src/views/Contract/Events/Event/event.js]: Address above not copyable?

File: js/src/modals/CreateAccount/AccountDetails/accountDetails.js [hunk 2]
```diff
@@ -31,7 +31,8 @@ export default class AccountDetails extends Component {
     return (
       <Form>
         <Input
-          disabled
+          readOnly
+          copiable
```
Comment (ngotchac) [inline js/src/modals/CreateAccount/AccountDetails/accountDetails.js]: Right

File: js/src/views/Contract/Events/Event/event.js (hunk 1 above)
Comment (ngotchac) [inline js/src/views/Contract/Events/Event/event.js]: Addresses are always copyable if `disabled`/`readonly`. Is that not right?

Comment (jacogr) [inline js/src/views/Contract/Events/Event/event.js]: Aaaahhhh... sorrry, must have had one eye closed. Perfect.

**Pull request merged**